include/dbstructure.php

```diff
@@ -918,7 +918,7 @@ function db_definition($charset) {
 					"ownerid_created" => array("owner-id","created"),
 					"wall_body" => array("wall","body(6)"),
 					"uid_visible_moderated_created" => array("uid","visible","moderated","created"),
-					"uid_uri" => array("uid","uri"),
+					"uid_uri" => array("uid", "uri"),
```

Comment:
Yeah. I wanted to make this index UNIQUE - but had the problems that I described elsewhere.